deps/build.jl

```diff
@@ -34,7 +39,7 @@ provides(SimpleBuild,
         CreateDirectory(joinpath(prefix, "lib"))
         @build_steps begin
             ChangeDirectory(srcdir)
-            `$make USE_DSFMT=1 DSFMT_includedir="$(joinpath(BinDeps.depsdir(libRmath),"dSFMT"))"
+            `$MAKE_CMD USE_DSFMT=1 DSFMT_includedir="$(joinpath(BinDeps.depsdir(libRmath),"dSFMT"))"
```

Comment:
Yeah, it just incorporates the "is this BSD? gmake, otherwise make" into BinDeps instead of reproducing it in each repo. Doesn't hurt to have the custom logic that's already here though.